.rubocop.yml

```diff
@@ -30,3 +30,7 @@ Style/IndentationConsistency:
 
 Style/RedundantReturn:
   Enabled: true
+
+Style/StringLiterals:
+  Enabled: true
+  EnforcedStyle: single_quotes
```

Comment:
Since we are enabling rubocop, and it runs quite fast, we should get in a habit of running it before every push. When we do, I don't expect many CI failures because of it. With autocorrect this is fixed in few seconds.